grpc/deny_friend_request.go

```diff
@@ -3,15 +3,14 @@ package grpc
 import (
 	"context"
 
-	database_wiiu "github.com/PretendoNetwork/friends-secure/database/wiiu"
+	database_wiiu "github.com/PretendoNetwork/friends/database/wiiu"
 	pb "github.com/PretendoNetwork/grpc-go/friends"
 )
 
 func (s *gRPCFriendsServer) DenyFriendRequest(ctx context.Context, in *pb.DenyFriendRequestRequest) (*pb.DenyFriendRequestResponse, error) {
-	// TODO - Make this return an error
-	database_wiiu.SetFriendRequestDenied(in.GetFriendRequestId())
+	err := database_wiiu.SetFriendRequestDenied(in.GetFriendRequestId())
```

Comment:
This should probably have a more robust error check, similar to `AcceptFriendRequest`